src/client/phy-ue.cc

```diff
@@ -541,6 +563,10 @@ void PhyUe::Start() {
               PrintPerFrameDone(PrintType::kDecode, frame_id);
               decode_counters_.Reset(frame_id);
 
+              if (kEnableCsvLog) {
```

Comment:
Remove this too.  Just let RecordBer and RecordSer (maybe make 1 function) do nothing if logging is disabled